README.md

```diff
@@ -1,6 +1,8 @@
 # Validating Admission Webhook for Service Binding
 
-A simple admission webhook which rejects every `ServiceBindingRequest`
+An validating admission controller which validates `ServiceBindingRequests` before they are created on the cluster.
```

Comment:
typo

`An validating` -> `A validating`

Reply:
Thank you!